samples/tab-stage-view/csharp/TabInStageView/README.md

```diff
@@ -0,0 +1,112 @@
+﻿This App talks about the Teams tab in stage view with CSharp.
```

Comment:
Could you please move this Read me file one level up to csharp folder? Please update the image paths accordingly.

Reply:
Updated